tests/fixtures/added-website.json

```diff
@@ -0,0 +1,187 @@
+      "newVersion": {
+        "type": "Feature",
+        "properties": {
+          "website": "http://example.com"
+        },
+        "geometry": {
+          "type": "Point",
+          "coordinates": [
+            76.85708999633789,
+            16.828531862816284
+          ]
+        }
+      },
+      "oldVersion": null,
+      "expectedResult": {
+        "result:added-website": true
+      }
+    },
+    {
+      "description": "test node without website",
+      "newVersion": {
+        "type": "Feature",
+        "properties": {},
+        "geometry": {
+          "type": "Point",
+          "coordinates": [
+            76.85708999633789,
+            16.828531862816284
+          ]
+        }
+      },
+      "oldVersion": null,
+      "expectedResult": {}
+    },
+    {
+      "description": "test adding the website to existing Feature",
+      "newVersion": {
+        "type": "Feature",
+        "properties": {
+          "website": "http://example.com"
+        },
+        "geometry": {
+          "type": "Point",
+          "coordinates": [
+            76.85708999633789,
+            16.828531862816284
+          ]
+        }
+      },
+      "oldVersion": {
+        "type": "Feature",
+        "properties": {},
+        "geometry": {
+          "type": "Point",
+          "coordinates": [
+            76.85708999633789,
+            16.828531862816284
+          ]
+        }
+      },
+      "expectedResult": {
+        "result:added-website": true
+      }
+    },
+    {
+      "description": "test modifying the website to existing Feature",
+      "newVersion": {
+        "type": "Feature",
+        "properties": {
+          "website": "http://example.com"
+        },
+        "geometry": {
+          "type": "Point",
+          "coordinates": [
+            76.85708999633789,
+            16.828531862816284
+          ]
+        }
+      },
+      "oldVersion": {
+        "type": "Feature",
+        "properties": {"website": "http://openstreetmap.org"},
+        "geometry": {
+          "type": "Point",
+          "coordinates": [
+            76.85708999633789,
+            16.828531862816284
+          ]
+        }
+      },
+      "expectedResult": {
+        "result:added-website": true
+      }
+    },
+    {
+      "description": "test delete website of existing feature",
+      "newVersion": {
+        "type": "Feature",
+        "properties": {},
+        "geometry": {
+          "type": "Point",
+          "coordinates": [
+            76.85708999633789,
+            16.828531862816284
+          ]
+        }
+      },
+      "oldVersion": {
+        "type": "Feature",
+        "properties": {"website": "http://openstreetmap.org"},
+        "geometry": {
+          "type": "Point",
+          "coordinates": [
+            76.85708999633789,
+            16.828531862816284
+          ]
+        }
+      },
+      "expectedResult": {
+        "result:added-website": true
```

Comment:
@amishas157 We thought about this a little and for this iteration wanted to flag even deletions of `website`.